scijava/scijava-ops-api/templates/main/java/org/scijava/ops/api/OpBuilder.vm

```diff
@@ -133,18 +133,50 @@ public class OpBuilder {
 	 */
 	public final class Arity0 {
 
+		/**
+		 * Match a {@link org.scijava.function.Computers} op that can use the given pre-allocated output instance.
```

Comment:
It's misleading to suggest that calling this method performs a match. It's building a request. The match happens later. Instead of "Match" I would say "Specifies" or "Requests" or "Indicates" or some such.